lib/dom.js

```diff
@@ -3263,3 +3263,25 @@ declare var status: string;
 declare var top: WindowProxy;
 declare function getSelection(): Selection | null;
 declare var customElements: CustomElementRegistry;
+
+/* Notification */
+declare class Notification extends EventTarget {
+  static permission: string;
```

Comment:
Maybe 

    type NotificationPermission = 'default' | 'denied' | 'granted';

and then change this to

    static permission: NotificationPermission;

? https://notifications.spec.whatwg.org/#enumdef-notificationpermission